ELK/apm-server.yml

```
@@ -0,0 +1,7 @@
+apm-server:
+  host: "0.0.0.0:8200" 
+
+output.elasticsearch:
+  hosts: ["https://elasticsearch:9200"]
+  api_key: "${ELASTICSEARCH_API_KEY}"
+  ssl.verification_mode: none
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)